test/scripts/delete-cluster.sh

```diff
@@ -14,10 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This shell script is used to build an image from our argo workflow
+# This shell script is used to delete a cluster.
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
-# TODO: deploy kfserving and run e2e use test cases
+CLUSTER_NAME="${CLUSTER_NAME}"
+ZONE="${GCP_ZONE}"
+PROJECT="${GCP_PROJECT}"
+
+echo "Activating service-account"
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+echo "Tearing down the cluster"
+gcloud container clusters delete ${CLUSTER_NAME} --zone=${ZONE} --project=${PROJECT} --async
```

Comment:
Let's create a regional cluster. Zonal clusters can be stuck reconciling (if masters are upgrading.) It doesn't cost any more.

Reply:
@ellis-bigelow Checked other project under Kubeflow (such as katib and metadata), seems all used `--zone`, and the "${ZONE}" is passed from the test framework, not familiar with google cloud, but may I suggest to use Zonal clusters to be consistent with others? Thanks.

Reply:
Hard disagree. Regional clusters cost the same amount and provide greater reliability. This feature came out after those tests were written.